packages/edit-site/src/components/global-styles/screen-revisions/style.scss

```diff
@@ -8,62 +8,80 @@
 	margin: 0;
 	li {
 		margin-bottom: 0;
-		border-left: 1px solid $gray-300;
 	}
 }
 
 .edit-site-global-styles-screen-revisions__revision-item {
 	position: relative;
-	padding: $grid-unit-10 0 $grid-unit-10 $grid-unit-15;
+	padding-left: $grid-unit-20;
+	overflow: hidden;
+	cursor: pointer;
 
-	&:first-child {
-		padding-top: 0;
+	&:hover {
+		background: rgba(var(--wp-admin-theme-color--rgb), 0.04);
+		.edit-site-global-styles-screen-revisions__date {
+			color: var(--wp-admin-theme-color);
+		}
 	}
 
-	&:last-child {
-		padding-bottom: 0;
+	&::before,
+	&::after {
+		position: absolute;
+		content: "\a";
+		display: block;
 	}
 
 	&::before {
 		background: $gray-300;
 		border-radius: 50%;
-		content: "\a";
-		display: inline-block;
 		height: $grid-unit-10;
 		width: $grid-unit-10;
-		position: absolute;
-		top: 50%;
-		left: 0;
+		top: $grid-unit-20 + 2;
+		left: $grid-unit-20 + 1; // So the circle is centered on the line.
 		transform: translate(-50%, -50%);
+		z-index: 1;
 	}
 	&.is-selected::before {
 		background: var(--wp-components-color-accent, var(--wp-admin-theme-color, #007cba));
 	}
-}
 
-.edit-site-global-styles-screen-revisions__revision-button {
-	width: 100%;
-	height: auto;
-	display: block;
-	padding: $grid-unit-10 $grid-unit-15;
+	&::after {
+		height: 100%;
+		left: $grid-unit-20;
+		top: 0;
+		width: 0;
+		border: 0.5px solid #ddd;
```

Comment:
Tiny nit: is it worth using `$gray-300` rather than `#ddd` specifically?

Reply:
It is worth it!